src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,66 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person as PersonType } from '../../types';
+
+type Props = {
+  person: PersonType,
+  findSlug: (name:string) => string | undefined,
```

Comment:
```suggestion
  findSlug?: (name:string) => string,
```
- you can do property optional by adding a question mark
- it's better to prepare people when you set them with mother/father, as for example, you need to pass the Person object to the `PersonLink` component